changelog.txt

```diff
@@ -47,6 +47,7 @@
 * Tweak - Add save payment method parameter to update intent call for non-deferred intent payment methods.
 * Fix - Checkout page focus loss
 * Fix - Updated payment method radio button selector to correctly find the selected payment method in different themes.
+* Update - Back button for tertiary level pages in settings page.
```

Comment:
At least for me, this was not too clear (without looking at the PR description).

Reply:
Made a slight change in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4191/commits/3e572f334d7e3fc3ef953d53320e89215cdb5537